data/sql/updates/pending_db_world/Fix_quest_11879.sql

```diff
@@ -0,0 +1,60 @@
+-- Fix quest 11879 "Kaw the Mammoth Destroyer"
+-- Add areatrigger
+DELETE FROM `areatrigger_scripts` WHERE `entry` = 4896;
+INSERT INTO `areatrigger_scripts` (`entry` , `ScriptName`) VALUES 
+(4896, 'SmartTrigger');
+-- Wooly Mammoth Bull SAI
```

Comment:
I don't see the AreaTrigger being used, is the script missing?

Reply:
there is a AreaTrigger there

Reply:
You can use it to trigger the event to start, then. It's way better that way. A Data Set on Creature Guid should work. There should be some examples in SAI for ATs